crates/bevy_ui/src/render/mod.rs

```diff
@@ -12,15 +12,15 @@ use bevy_app::prelude::*;
 use bevy_asset::{load_internal_asset, AssetEvent, Assets, Handle, HandleUntyped};
 use bevy_core::FloatOrd;
 use bevy_ecs::prelude::*;
-use bevy_math::{const_vec3, Mat4, Vec2, Vec3, Vec4Swizzles};
+use bevy_math::{const_vec3, Mat4, Vec2, Vec3, Vec3Swizzles, Vec4, Vec4Swizzles};
 use bevy_reflect::TypeUuid;
 use bevy_render::{
     camera::ActiveCameras,
     color::Color,
     render_asset::RenderAssets,
     render_graph::{RenderGraph, SlotInfo, SlotType},
     render_phase::{sort_phase_system, AddRenderCommand, DrawFunctions, RenderPhase},
-    render_resource::*,
+    render_resource::{std140::AsStd140, *},
```

Comment:
That shouldn't be needed.

Reply:
I'd love to remove it because I've been having problems with it because it aligns `[f32; 4]` to 16 bytes per float, so I had to use a Vec4. However, it's required by `DynamicUniformVec` because it's defined as `DynamicUniformVec<T: AsStd140>`, so it seems that I can't remove it.

Reply:
Ah, I meant "the import is bizarre". * imports everything in the namespace, so unless it's clashing somewhere the import can be simplified.

Reply:
it's importing everything from `render_resource::*` plus everything from `render_resource::std140` right? If I just leave the star then I need to use `#[Derive(std140::AsStd140]` in the code I think. Or am I missing something here?

Reply:
> If I just leave the star then I need to use #[Derive(std140::AsStd140)] in the code I think

Ah, okay! If that's the case, then I'm the one who's mistaken then. Double check removing it, but if that's true then it's fine.